cmd/controller/main.go

```diff
@@ -433,6 +444,55 @@ type httpServer struct {
 	http.ServeMux
 }
 
+func runLeaderElection(ctx context.Context, cancel context.CancelFunc, logger *logrus.Entry, ctlConf config, kubeClient *kubernetes.Clientset, start func(_ context.Context)) {
+	if !ctlConf.LeaderElection {
```

Comment:
nit: I would just pass in the boolean, maybe as `doLeaderElection`.

Reply:
Done!